docs/articles/documentation/getting-started/index.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: docs
+title: Getting Started
+permalink: /documentation/getting-started/
+---
+# Getting Started
+
+This guide provides step-by-step instructions on how to create a functional web test with TestCafe and contains the following sections.
+
+* [Installing TestCafe](#installing-testcafe)
+* [Writing Test](#writing-test)
+  * [General Code Structure](#general-code-structure)
+  * [Performing Actions on Page](#performing-actions-on-page)
+  * [Observing Page State](#observing-page-state)
+  * [Assertions](#assertions)
+* [Running Test](#running-test)
+* [Viewing Test Results](#viewing-test-results)
+
+## Installing TestCafe
+
+You can install TestCafe into a website project you are going to test.
+
+First, make sure that [Node.js](https://nodejs.org/) and [npm](https://www.npmjs.com/) are installed on your computer. Then go to the tested project directory and run a single command -
+`npm install --save-dev testcafe`.
+
+This tutorial will demonstrate how to test not a website project, but the published web page - [http://testcafe.devexpress.com/Example](http://testcafe.devexpress.com/Example).
+To create tests for this page, create the *my-project* directory anywhere on your computer and install TestCafe into it. You can do this from a command line.
+
+```bash
+mkdir my-project
+cd my-project
+npm install --save-dev testcafe
+```
+
+## Writing Test
+
+You can organize your tests into any directory on your computer. It is a good practice to create the *test* directory as a subdirectory of the website project you are going to test and place tests there.
+
```

Comment:
Also it's worth mention that you can put test file anywhere and pass their path to testcafe